superset/db_engine_specs/presto.py

```diff
@@ -653,12 +656,15 @@ def expand_data(  # pylint: disable=too-many-locals
                 # expand columns; we append them to the left so they are added
                 # immediately after the parent
                 expanded = get_children(column)
-                to_process.extendleft((column, level) for column in expanded)
+                to_process.extendleft((column, level) for column in expanded[::-1])
```

Comment:
I reversed `expanded` so that sub-columns are added in the right order, otherwise we get:

```
a ROW(b, c) => a, a.c, a.b
```